build.gradle.kts

```diff
@@ -348,7 +348,8 @@ extra["compilerArtifactsForIde"] = listOf(
 
 // TODO: fix remaining warnings and remove this property.
 extra["tasksWithWarnings"] = listOf(
-    ":kotlin-gradle-plugin:compileKotlin"
+    ":kotlin-gradle-plugin:compileKotlin",
+    ":compiler:fir:checkers:compileKotlin"
```

Comment:
?

Reply:
This is added just for my convenience, for a while. Of course, it should be removed